src/minecraft_monitor.py

```diff
@@ -31,25 +31,12 @@ def __init__(self, config_file: Union[str, Path], debug_mode=False):
             logging.FileHandler(self.config['log_file'])
         )
 
-    def parse_command(self, command: str):
-        command_words = command.split()
-        if "start" in command_words:
-            if self.server_running:
-                return "Server started successfully."
-            else:
-                self.logger.error("Server failed to start")
-                return "Error: server did not start successfully."
-        elif "stop" in command_words:
-            self.shutdown_game_server()
-            return "Server has shutdown."
-        elif "restart" in command_words:
+    def parse_custom_commands(self, command_words: List[str]):
+        if "restart" in command_words:
```

Comment:
This should be re-start.
```suggestion
        if "re-start" in command_words:
```

Reply:
This isn't necessary.